CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Changed
 - Laravel recipe should not run `artisan:cache:clear` in `deploy` task
 - Pass-through the quiet mode into the git commands for updating code
+- Updated .gitignore
```

Comment:
I think this is chore update, no users to inform.

Reply:
OK. I'll revert this change. Thank you.